src/Cli/OrgCommands.php

```diff
@@ -226,6 +228,8 @@ public function orgUpdateProjectsInfo($csv_file, $options = [
                             $this->logger->warning(sprintf("Skipping project %s because of empty or invalid support level.", $projectFullName));
                             continue;
                         }
+                    } elseif ($projectSupportLevel === 'Unsupported') {
```

Comment:
If we don't combine unsupported and deprecated into a single support level, this conditional should probably also use the unsupported text for deprecated projects

Reply:
Done